modules/deploy/pages/setting-up-dr-cluster.adoc

```diff
@@ -35,38 +36,73 @@ include::ROOT:partial$_show_page_header_block.adoc[]
 == Introduction
 
 {server-xdcr--xref} (XDCR) replicates data between two or more autonomous Couchbase Server clusters.
-It serves an important role in supporting Disaster Recovery (DR) and Data Migration, even where Sync Gateway is the normal replicator of choice for mobile data.
+It plays an important role in supporting Disaster Recovery (DR) and Data Migration, even where Sync Gateway is the normal replicator of choice for mobile data.
 
 
 == Recommended Deployment Models
 
+
+=== Zero Downtime Active-Active Disaster Recovery
+// tag::zero-downtime-active-active[]
+
+This model provides true zero-downtime disaster recovery using bi-directional XDCR between two active mobile clusters.
+Both clusters remain operational, with seamless fail-over through load balancer switching.
+`shared_bucket_access=true` becomes required when loading a database in SGW 4.0+, and you must configure both clusters with `import_docs=true`.
```

Comment:
I think better than mentioning `shared_bucket_access` here, mention at the top that this requires running Sync Gateway 4.0+ on both sides of the active-active XDCR setup.

Reply:
Updated! I moved the Sync Gateway 4.0+ requirement to the top of the section and removed the specific `shared_bucket_access` configuration detail as suggested.